implementations/rust/ockam/ockam/src/entity/issuer.rs

```diff
@@ -0,0 +1,123 @@
+use crate::{
+    Address, CredentialAttribute, CredentialFragment2, CredentialIssuer, CredentialOffer,
+    CredentialRequest, CredentialSchema, OckamError, OfferIdBytes, ProfileTrait, ProofBytes,
+    PublicKeyBytes, Result, Route,
+};
+
+pub struct LocalIssuingEntity<E: ProfileTrait> {
```

Comment:
Maybe we're ready to rename ProfileTrait -> EntityTrait

Reply:
@mrinalwadhwa do you think it's time?